10.9-libcxx/stable/main/finkinfo/libs/perlmods/libwww-pm.info

```diff
@@ -2,7 +2,7 @@
 Info2: <<
 Package: libwww-pm%type_pkg[perl]
 Version: 6.71
-Revision: 1
+Revision: 2
```

Comment:
Not sure why this is revved up. But Hanspeter pushed an update that includes 5.34.1 so no need to do anything here anymore.